main.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Francisco Souza. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/fsouza/fake-gcs-server/fakestorage"
+)
+
+func generateObjectsFromFiles() []fakestorage.Object {
+	objects := []fakestorage.Object{}
+
+	// if a /data volume is mounted in the container
+	if _, err := os.Stat("/data"); !os.IsNotExist(err) {
```

Comment:
I liked the config refactor of PR #46, making the paths parameters could be part of that near future update. It may be cleaned to skip the initial load if the path is not defined.